src/features/company/components/Company.tsx

```diff
@@ -3,31 +3,44 @@ import { Peers } from "../../peers";
 import { useSelector } from "react-redux";
 import { Loading } from "../../loading";
 import "./Company.css";
+import { AppState } from "../../../store";
 
 export const Company = () => {
   const { selectedCompanyOverview } = useSelector(
-    state => state.companyOverviewData
+    (state: AppState) => state.companyOverviewData
   );
-  const { selectedTopPeers } = useSelector(state => state.peerData);
+  const { selectedTopPeers } = useSelector((state: AppState) => state.peerData);
 
   const renderCompanyComponent = () => (
     <>
       <div className="company-overview__title">
-        {selectedCompanyOverview.companyName || "N/A"} (
-        {selectedCompanyOverview.symbol || "N/A"})
+        {(selectedCompanyOverview !== null &&
+          selectedCompanyOverview.companyName) ||
+          "N/A"}{" "}
```

Comment:
As mentioned to me by Kyle, the empty string `{" "}` can be replaced with a string literal, so:
 {`${(selectedCompanyOverview && selectedCompanyOverview.companyName) ||
          "N/A"} (${(selectedCompanyOverview &&
          selectedCompanyOverview.symbol) ||
          "N/A"})`}